react-form-controls/my-app/src/RegistrationFormControlled.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+
+export default function FormControlled() {
+  const [userName, setUserName] = useState('');
+  const [password, setPassword] = useState('');
+
+  return (
+    <form
+      onSubmit={(e) => {
+        e.preventDefault(),
+          setUserName(userName),
+          setPassword(password),
+          console.log(userName, password);
```

Comment:
To get an object output in the log, just wrap the two fields in a literal object:
`{ userName, password }`.